appengine/standard_python3/bundled-services/blobstore/django/README.md

```diff
@@ -0,0 +1,5 @@
+# Access legacy bundled services for Python 3
```

Comment:
Should this say anything specific for Django?

Reply:
Update the links to the corresponding page in our public doc. I don't want to repeat the doc here, since they will easily go out of sync.